include/sparrow/types/data_traits.hpp

```diff
@@ -68,6 +68,14 @@ namespace sparrow
         using const_reference = std::string_view;
     };
 
+    template <>
+    struct arrow_traits<std::string_view>
+    {
+        static constexpr data_type type_id = data_type::STRING_VIEW;
+        using value_type = std::string_view;
+        using const_reference = std::string_view;
+    };
```

Comment:
Same for the traits, they should be defined for types participating in the value_type variant, not types participating in the const_reference variant.